omnibus/files/private-chef-ctl-commands/rotate_credentials.rb

```diff
@@ -122,6 +137,77 @@
   exit(0)
 end
 
+add_command_under_category "require-credential-rotation", "credential-rotation", "Disable the Chef Server and require credential rotation", 2 do
```

Comment:
Stopping the Chef Server and disabling the services are two different things and this command technically does both. It does disable the Chef Server in that runsvdir will no longer start any services which effectively renders them disabled. systemd and runit use that word to describe what we're doing here.  Perhaps I should say "safely disable the chef server services" or something to that effect?

As per a different file, I could do that but there this command and rotating-shared-secrets share some common code paths.